src/js/components/Notifications/Notifications.react.js

```diff
@@ -21,13 +21,22 @@ export default class Notifications extends Component {
         this.state = {};
     }
 
-    render() {
-        // TODO (y.solovyov): should enforce reporting of this kind of long lines through eslint
-        const notifications = this.props.notifications.map((notification, index) => <Notification type={ notification.type } content={ notification.content } key={ index } />);
+    getNotifications() {
+        return this.props.notifications.map((notification, index) => {
+            return (
+                <Notification
+                    type={ notification.type }
+                    content={ notification.content }
+                    key={ index }
+                />
+            );
+        });
+    }
 
+    render() {
         return (
             <div className='notifications'>
-                { notifications }
+                { this.getNotifications() }
```

Comment:
Not really usefull, and it moves some logic outside of render(), please roll back ;)